packages/testing/src/api/index.ts

```diff
@@ -36,6 +39,10 @@ export function apps(): (admin.app.App | null)[] {
   return admin.apps;
 }
 
+export function firestoreApps(): (FirebaseApp | null)[] {
```

Comment:
The distinction between `Admin App`, `Test App` and `Firestore Test App` is not very clear.  Would it be possible to add an argument to `initializeTestApp` that allows us to turn on Firestore support?